server/models/server-service.js

```diff
@@ -115,4 +115,20 @@ module.exports = function(ServerService) {
     ServerService.app.serviceManager.getDeployment(this, ctx.req, ctx.res);
   }
   ServerService.prototype.getPack = getPack;
+
+  ServerService.prototype.setEnv =
```

Comment:
Eww.. :-P

Looks like I also missed the style change (expression to statement) when I rebased.